packages/components/transitions/tests/slide-fade.test.tsx

```diff
@@ -1,38 +1,121 @@
-import { render } from "@yamada-ui/test"
+import { waitFor } from "@testing-library/react"
```

Comment:
`waitFor` can be called from `@yamada-ui/test`.

Reply:
@setodeve

Fixed!